### PR TITLE
use correct getter; user could be Web or Mobile user

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

**Comment by sravfeyn** [comment]
Introduced here https://github.com/dimagi/commcare-hq/pull/22918/files

**Review by czue — APPROVED** [review]

File: corehq/apps/users/models.py [hunk 1]
```diff
@@ -2476,9 +2476,6 @@ def get_sql_locations(self, domain=None):
         else:
             return SQLLocation.objects.none()
 
-    def get_location(self, domain):
```
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/users/models.py]: Yeah, there wasn't a use except the one above

**Comment by czue** [comment]
should this be merged @sravfeyn ?

**Comment by sravfeyn** [comment]
Oh yes, I will resolve conflicts and do that. Thanks!

**Comment by calellowitz** [comment]
I think this was resolved by reverting the original PR and this hotfix can be closed

**Pull request closed without merging**